pkgs/by-name/en/envision/envision-unwrapped.nix

```diff
@@ -0,0 +1,86 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, appstream-glib
+, cargo
+, meson
+, ninja
+, pkg-config
+, rustPlatform
+, rustc
+, wrapGAppsHook4
+, cairo
+, desktop-file-utils
+, gdb
+, gdk-pixbuf
+, glib
+, gtk4
+, gtksourceview5
+, libadwaita
+, libgit2
+, libusb1
+, openssl
+, pango
+, vte-gtk4
+, zlib
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "envision-unwrapped";
+  version = "0-unstable-2024-06-16";
+
+  src = fetchFromGitLab {
+    owner = "gabmus";
+    repo = "envision";
+    rev = "4c42a1c71cf51f83dec422178937f293210dfd07";
+    hash = "sha256-76qAWV7wPpguguuEQ9941vQsBOUF+LJ5mFVI0Nl6Dr0=";
+  };
+
+  cargoDeps = rustPlatform.importCargoLock {
+    lockFile = finalAttrs.src + "/Cargo.lock";
+    outputHashes = {
+      "libmonado-rs-0.1.0" = "sha256-PsNgfpgso3HhIMXKky/u6Xw8phk1isRpNXKLhvN1wIE=";
+    };
+  };
+
+  nativeBuildInputs = [
+    appstream-glib
+    cargo
+    meson
+    ninja
+    pkg-config
+    rustPlatform.cargoSetupHook
+    rustc
+    wrapGAppsHook4
+  ];
+
+  buildInputs = [
+    cairo
+    desktop-file-utils
+    gdk-pixbuf
+    glib
+    gtk4
+    gtksourceview5
+    libadwaita
+    libgit2
+    libusb1
+    openssl
+    pango
+    vte-gtk4
+    zlib
+  ];
+
+  postInstall = ''
+    wrapProgram $out/bin/envision \
+      --prefix PATH : "${lib.makeBinPath [gdb]}"
+  '';
+
+  meta = with lib; {
+    description = "UI for building, configuring and running Monado, the open source OpenXR runtime";
+    homepage = "https://gitlab.com/gabmus/envision";
+    license = licenses.agpl3Only;
+    mainProgram = "envision";
+    maintainers = with maintainers; [ pandapip1 Scrumplex ];
```

Comment:
CC @Scrumplex. Would you like to be listed as a maintainer?

Reply:
yup ^^